stdlib/experimental/http/get_test.go

```diff
@@ -92,3 +93,32 @@ http.get(url:"http://127.1.1.1/path/a/b/c", headers: {x:"a",y:"b",z:"c"})
 		t.Errorf("unexpected cause of failure, got err: %v", err)
 	}
 }
+
+func TestGet_Timeout(t *testing.T) {
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, request *http.Request) {
+		var err error
+		if err != nil {
+			w.WriteHeader(500)
+			w.Write([]byte(err.Error()))
+		}
+		time.Sleep(time.Second * 8)
+		w.WriteHeader(204)
+	}))
+	defer ts.Close()
+
+	script := fmt.Sprintf(`
+import "experimental/http"
+
+resp = http.get(url:"%s/path/a/b/c", timeout: 4s)
```

Comment:
Can you make this timeout something really small? That way the test still runs fast. Something like `10ms` or maybe `50ms`. It will timeout really fast but that is ok for the test.

Reply:
Great idea. Done.